Makefile

```diff
@@ -56,7 +56,7 @@ test: generate fmt vet manifests
 	go test ./pkg/... ./cmd/... -coverprofile cover.out
 
 .PHONY: kubeflare
-kubeflare: generate fmt vet bin/kubeflare 
+kubeflare: clean-and-tidy generate fmt vet bin/kubeflare 
```

Comment:
Should we clean before building each time? This seems like it might be covering something else up?

Reply:
ah, added that while i was troubleshooting. meant to remove that!